README.rst

```diff
@@ -35,15 +32,28 @@ Getting started
 
 2. Add ``wagtailtrans`` to your INSTALLED_APPS
 
-3. Perform a migration
+3. Add `WAGTAILTRANS_TEMPLATE_DIR` (from wagtailtrans import WAGTAILTRANS_TEMPLATE_DIR) to your template dirs setting
+
+.. code-block:: python
+    from wagtailtrans import WAGTAILTRANS_TEMPLATE_DIR
+
+    TEMPLATES = [{
+        # ....
+        'DIRS': [
+            WAGTAILTRANS_TEMPLATE_DIRS,
```

Comment:
This is a seperate issue, better to handle it in a different issue than this PR.